From 28ce2545e1951b7a2b8b04637b522d55028d4a2a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Aug 2021 21:00:23 +0300
Subject: [PATCH] Revert "[SW-51308] gaudi: security: allow pcie gw access"

Another w/a was found and we no longer need the host access the
PCIe GW.
Revert the change allowing the access.

This reverts commit a77ea23212f8bb755361cc5cf477bfaac2f77bd0.

Change-Id: Ide6a1d10f1d7f12d0ee4b6705c88e17266fa5d8a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/security.c b/arch/gaudi/security.c
index 77b740f..99cdb48 100644
--- a/arch/gaudi/security.c
+++ b/arch/gaudi/security.c
@@ -848,18 +848,7 @@
 		PROT_BIT(mmPCIE_WRAP_RR_ELBI_RD_SEC_REG_CTRL));
 	regmap_writel(mask, PRIV_PROT_REG(prot_base));
 
-	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x280 - 0x300 */
-	set_prot_bits_range_all(prot_base, SNGL_REG_PROT_SIZE);
-
-	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x300 - 0x380 */
-	mask = PROT_BIT(mmPCIE_WRAP_LBW_GW_ADDR_0 - sizeof(u32));
-	regmap_writel(mask, PRIV_PROT_REG(prot_base));
-
-	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x380 - 0x400 */
-	mask = ~PROT_BIT(mmPCIE_WRAP_LBW_GW_STATUS_7);
-	regmap_writel(mask, PRIV_PROT_REG(prot_base));
-
-	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x400 - 0xF80 */
+	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x280 - 0xF80 */
 	set_prot_bits_range_all(prot_base, PROT_SIZE_MAX - (prot_base - base));
 }
 
